app/render/audioparams.h

```diff
@@ -75,6 +78,9 @@ class AudioParams {
   {
     set_default_footage_parameters();
     timebase_ = sample_rate_as_time_base();
+
+    // Cache channel count
+    channel_count_ = av_get_channel_layout_nb_channels(this->channel_layout());
```

Comment:
You'll also want to set this under `set_channel_layout()`, and it'd probably be best to put this under a new function like `calculate_channel_count` to minimize on copy/pasted code (see `VideoParams:: calculate_effective_size`).

Reply:
Will do, thanks.